frontend/express/public/javascripts/pre-login.js

```diff
@@ -1,4 +1,4 @@
-/*global store, jQuery, $, document, countlyGlobal, filterXSS */
+/*global store, jQuery, $, document, countlyGlobal, filterXSS, alert, window */
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['window' is defined but never used.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=4805903)